src/pretix/base/views/metrics.py

```diff
@@ -3,31 +3,31 @@
 from .. import metrics
 
 
-def unauthedResponse():
+def unauthed_response():
     content = "<html><title>Forbidden</title><body>You are not authorized to view this page.</body></html>"
-    unauthedResponse = HttpResponse(content, mimetype="text/html")
-    unauthedResponse["WWW-Authenticate"] = 'Basic realm="metrics"'
-    unauthedResponse.status_code = 401
+    response = HttpResponse(content, mimetype="text/html")
+    response["WWW-Authenticate"] = 'Basic realm="metrics"'
+    response.status_code = 401
```

Comment:
indeed, thanks for the hint. after fixing that I should probably stop coding for today...